lib/util.js

```diff
@@ -211,18 +212,30 @@ util.getRequestBodyObj = function(allowedProps,args) {
 var _requestHelper = function(options,done) {
 
     request(options, function (err, response, payload) {
-
-        if (!err) {
-            //console.log(payload);
-        }
-        else {
+        if (err) {
             console.log('api call error');
             console.log(err);
         }
 
-        done(err, payload);
+        done(err, payload, parseRateLimits(response.headers));
     });
 };
+
+function parseRateLimits(headers) {
+    if(!headers[util.rateLimit] || !headers[util.rateUsage]) {
+        return null;
+    }
+
+    var limit = headers[util.rateLimit].split(',')
+        , usage = headers[util.rateUsage].split(',');
+
+    return {
+        shortTermUsage: parseInt(usage[0]),
```

Comment:
`parseInt()` requires a second parameter `radix`. This may or may not default to the expected value 10 depending on the implementation, but is not guaranteed. Could you please add this?

See https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/parseInt for details.